templates/Lambda4AppSyncHTTP/package.json

```diff
@@ -8,6 +8,7 @@
   },
   "author": "",
   "license": "ISC",
+  "type": "module",
```

Comment:
Kris mentioned that you should be able to revert the renaming of `js` files to `mjs` can be potentially be reverted after this change.